opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreationFromEntity.jsx

```diff
@@ -770,14 +770,19 @@ const StixCoreRelationshipCreationFromEntity = (props) => {
         sx={{ zIndex: 1202 }}
         classes={{ paper: classes.drawerPaper }}
         onClose={handleClose}
+        SlideProps={{
+          // containerRef is forwarded to ListLinesContent so it listens to scroll events and load data with an InfiniteLoader
+          // we must target the element inside the Drawer that holds the scrolling = the Slide
+          ref: containerRef,
+        }}
```

Comment:
@Kedae this is the proper solution

MUI doc is clumsy but when you read between the lines...